<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFriendDialog</class>
 <widget class="QDialog" name="AddFriendDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>668</width>
     <height>486</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>添加好友</string>
    </attribute>
    <widget class="QWidget" name="widget_friendInfo" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>501</width>
       <height>271</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>73</x>
        <y>50</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 账号:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_friend_account">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>73</x>
        <y>90</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 昵称：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_friendName">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_reqInfo">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>验证信息:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_addFriend">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>添加好友</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>账号:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_account"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_find">
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>添加群聊</string>
    </attribute>
    <widget class="QWidget" name="widget_friendInfo_2" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>501</width>
       <height>271</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>73</x>
        <y>50</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>群组号:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_group_account">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>73</x>
        <y>90</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 群组名：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_groupName">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_reqInfo_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>验证信息:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_addGroup">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>添加群组</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>群组账号:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_account_2"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_findGroup">
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>创建群组</string>
    </attribute>
    <widget class="QWidget" name="widget_friendInfo_3" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>501</width>
       <height>291</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>73</x>
        <y>90</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 群组名：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_createName">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_createDesc">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>群组描述:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_createGroup">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>添加群组</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
